src/utils/uuid.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import crypto from 'crypto';
```

Comment:
will it work in the browser?

Reply:
Yes, importing `crypto` always works. It's only a subset of the `crypto` methods that are not available. You can see this by yourself or in the CI logs by verifying that `test_preload_scripts.py` passes.

Reply:
I think it might work due to some rollup optimizations but this does not generally work in the browser.

<img width="809" alt="Screenshot 2023-05-22 at 13 40 01" src="https://github.com/GoogleChromeLabs/chromium-bidi/assets/399150/2dfe047f-44f6-4f7b-97cb-fdb195a55ad9">

Reply:
Yes, we have

```
    globals: {
      crypto: 'crypto',
    },
```

in our rollup config from a previous PR. If I remove it, it will probably fail. Let me try it.

Reply:
Confirmed: Upon removing this config from rollup, `import crypto` ceases to work (it times out the E2E tests).

Upon adopting your suggestion with `globalThis`, it works again.

Done.